MAINTAINERS.md

```diff
@@ -0,0 +1,161 @@
+# Maintainers
+
+This document serves to clarify the responsibilities of a maintainer as well as
+serve as a wiki for maintainers to link to / include resources.
+
+
+## Current Maintainers
+
+The current maintainers are:
+
+| name | focus areas |
+------------------
+|   @achow101  | wallet, PSBTs |
+|   @fanquake | general, tooling, build system |
+|   @hebasto | UI/UX |
+|   @laanwj | general |
+|   @marcofalke | general, tests, QA |
+|   @glozow | general, transaction relay and mempool policy |
```

Comment:
Fixed the table markdown and changed the order of maintainers according to [dates](https://bitcoin.stackexchange.com/questions/176/is-there-a-list-of-core-bitcoin-committers) 

```suggestion
| Name  | Focus Areas |
| ------------- | ------------- |
| @laanwj | General  |
| @MarcoFalke   | General, tests, QA  |
| @fanquake   | General, tooling, build system  |
| @hebasto    | UI/UX  |
| @achow101    | Wallet, PSBTs  |
| @glozow     | General, transaction relay and mempool policy |
```